geomstats/numerics/geodesic.py

```diff
@@ -62,26 +71,33 @@ class ExpODESolver(ExpSolver):
     """
 
     def __init__(self, integrator=None):
+        super().__init__(solves_ivp=False)
+
         if integrator is None:
             integrator = GSIVPIntegrator()
 
+        self._integrator = None
         self.integrator = integrator
 
+    @property
+    def integrator(self):
+        """An instance of ODEIVPSolver."""
```

Comment:
should we call it "solver" then? it would help readability, if Solver-type objects are indeed called solvers.

Reply:
Bad naming here. I probably need to call `ODEIVPSolver` -> `ODEIVPIntegrator` instead, since this object `.integrate`.